apigee_m10n.module

```diff
@@ -52,9 +52,35 @@ function apigee_m10n_theme($existing, $type, $theme, $path) {
       'variables' => ['api_product' => NULL],
     ],
     'rate_plan_detail' => [
-      'variables' => ['detail' => NULL],
+      'variables' => [
+        'detail' => NULL,
+        'ratecard_rates' => NULL,
+        'revshare_rates' => NULL,
+      ],
       'template' => 'rate-plan-detail',
     ],
+    'ratecard_rates' => [
+      'variables' => [
+        'rates' => NULL,
+      ],
+    ],
+    'revshare_rates' => [
+      'variables' => [
+        'rates' => NULL,
+      ],
+    ],
+    'rate_plan_detail__revshare' => [
+      'base hook' => 'rate_plan_detail',
+    ],
+    'rate_plan_detail__ratecard' => [
+      'base hook' => 'rate_plan_detail',
+    ],
+    'rate_plan_detail__revshare_ratecard' => [
+      'base hook' => 'rate_plan_detail',
+    ],
+    'rate_plan_detail__adjustable_notification' => [
+      'base hook' => 'rate_plan_detail',
```

Comment:
I agree, I'll left these in until I could confirm they won't be needed. I think the only thing that needs to be customized is "Rate card" vs "Rate card and Revenue" so that can just be done in the template.

Reply:
They have been removed.